docs/src/main/paradox/elasticsearch.md

```diff
@@ -111,17 +111,18 @@ Java
 Sinks and flows are configured with `ElasticsearchWriteSettings`.
 
 Scala
-: @@snip [snip](/elasticsearch/src/test/scala/docs/scaladsl/ElasticsearchSpec.scala) { #sink-settings }
+: @@snip [snip](/elasticsearch/src/test/scala/docs/scaladsl/ElasticsearchConnectorBehaviour.scala) { #sink-settings }
 
 Java
 : @@snip [snip](/elasticsearch/src/test/java/docs/javadsl/ElasticsearchTest.java) { #sink-settings }
 
 
-| Parameter           | Default | Description                                                                                            |
-| ------------------- | ------- | ------------------------------------------------------------------------------------------------------ |
-| bufferSize          | 10      | Flow and Sink batch messages to bulk requests when back-pressure applies.                             |
-| versionType         | None    | If set, `ElasticsearchSink` uses the chosen versionType to index documents. See [Version types](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#_version_types) for accepted settings. |
-| retryLogic | No retries | See below |
+| Parameter           | Default    | Description                                                                                            |
+| ------------------- | -------    | ------------------------------------------------------------------------------------------------------ |
+| bufferSize          | 10         | Flow and Sink batch messages to bulk requests when back-pressure applies.                             |
+| versionType         | None       | If set, `ElasticsearchSink` uses the chosen versionType to index documents. See [Version types](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html#_version_types) for accepted settings. |
+| retryLogic          | No retries | See below |
+| apiVersion          | V5         | See [supported versions](/elasticsearch/src/main/java/akka/stream/alpakka/elasticsearch/ApiVersion.java) |
```

Comment:
The link to the source won't work in the docs. We might need to live with a potential mismatch here. State the available API versions.